CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 #### Features
 
 * Your contribution here.
+* [#1795](https://github.com/ruby-grape/grape/pull/1795): Fix Accept header parsing bug - [@bschmeck](https://github.com/bschmeck)
```

Comment:
Missing a period at the end. It could also be more specific: "Fix: vendor/subtype parsing in an invalid Accept header"